src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,19 @@
-export const Tabs = () => {};
+export const Tabs = ({ tab, selectedTabId, onTabSelected, setTabItem }) => (
+  <li
+    data-cy="Tab"
+    className={`${selectedTabId === tab.id && 'is-active'}`}
+  >
+    <a
+      onClick={() => {
```

Comment:
It's a good practice to prevent the default action of the anchor tag in the onClick handler to avoid reloading the page.